src/app/pages/jails/jail-list/jail-list.component.ts

```diff
@@ -116,20 +116,26 @@ export class JailListComponent implements OnInit {
       enable: true,
       ttpos: "above",
       onClick: (selected) => {
-        let selectedJails = this.getSelectedNames(selected);
-        this.loader.open();
+        const selectedJails = this.getSelectedNames(selected);
+        this.snackBar.open(T('Updating selected plugins.'), 'close', { duration: 5000 });
         this.entityList.busy =
           this.ws.job('core.bulk', ["jail.update_to_latest_patch", selectedJails]).subscribe(
             (res) => {
-              this.loader.close();
-              if (res.state == 'SUCCESS') {
-                this.snackBar.open(T("Selected Jail(s) updated successfully."), T("Close"), { duration: 5000 });
+              let message = "";
+              for (let i = 0; i < res.result.length; i++) {
+                if (res.result[i].error != null) {
+                  message = message + '<li>' + selectedJails[i] + ': ' + res.result[i].error + '</li>';
+                }
+              }
+              if (message === "") {
+                this.snackBar.open(T('Selected jails updated.'), 'close', { duration: 5000 });
               } else {
-                new EntityUtils().handleWSError(this.entityList, res, this.dialogService);
+                message = '<ul>' + message + '</ul>';
+                this.dialogService.errorReport(T('Update of these jail failed'), message);
               }
             },
             (res) => {
-              this.loader.close();
+              this.snackBar.open(T('Updating selected jails failed'), 'close', { duration: 5000 });
```

Comment:
Add a period to match other snackbar messages, which are full sentences.
```suggestion
              this.snackBar.open(T('Updating selected jails failed.'), 'close', { duration: 5000 });
```